dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtilTest.java

```diff
@@ -795,10 +799,38 @@ public void testOnePullRequestForMultipleDockerfilesInSameRepo() throws Exceptio
 
         // Both Dockerfiles modified
         verify(dockerfileGitHubUtil, times(2))
-                .modifyOnGithub(any(), eq("image-tag"), eq("image"), eq("tag"), anyString());
+                .modifyOnGithub(any(), eq("image-tag"), eq("image"), eq("tag"), anyString(), anyString());
 
         // Only one PR created on the repo with changes to both Dockerfiles.
         verify(dockerfileGitHubUtil, times(1)).createPullReq(eq(parentRepo),
                 eq("image-tag"), eq(forkedRepo), any());
     }
+
+    @Test
+    public void testDockerfileWithIgnorePRComment() {
+        gitHubUtil = mock(GitHubUtil.class);
+        dockerfileGitHubUtil = new DockerfileGitHubUtil(gitHubUtil);
+        StringBuilder stringBuilder = new StringBuilder();
+        String img = "image";
+        String tag = "changed";
+
+        dockerfileGitHubUtil.changeIfDockerfileBaseImageLine(img, tag, stringBuilder, "FROM image:original # no-dfiu", "");
```

Comment:
Why not user DataProviders for these?

Reply:
Updated to use DataProviders. Thanks for suggesting.